### PR TITLE
Cart

### PR DESCRIPTION
**Pull request by saifudinbadani**
Screen Shot 2022-02-14 at 4 55 19 pm

### CODE REVIEW

File: css/cart.css [hunk 1]
```diff
@@ -0,0 +1,14 @@
+@import "../common.css";
+
+.cart-container{
```
Comment (tushargoyal812) [inline css/cart.css:3]: Suggestion: Use space before { it looks good, use beautify extension in vs code

**Review by tushargoyal812 — COMMENTED** [review]
just take care of suggestions and good to merge

File: css/cart.css (hunk 1 above)
Comment (saifudinbadani) [reply css/cart.css:3]: Sure. I will apply it.

**Review by palash23jain — APPROVED** [review]
color can be in same format for consistency. But it is not breaking change. Good to go as of now

**Pull request merged**